src/library/CMakeLists.txt

```diff
@@ -90,6 +90,11 @@ target_link_libraries(commissioner
         commissioner-common
 )
 
+target_compile_definitions(commissioner
+    PRIVATE
+        $<IF:$<BOOL:${OT_COMM_CCM}>, OT_COMM_CCM_ENABLE=1, OT_COMM_CCM_ENABLE=0>
```

Comment:
why not `OT_COMM_CONFIG_CCM_ENABLE`, following config macro naming convention of `openthread` ?

Reply:
okay to me, will do.